sdk/core/core-tracing/package.json

```diff
@@ -24,8 +24,8 @@
     "integration-test:browser": "echo skipped",
     "integration-test:node": "echo skipped",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
-    "lint:fix": "eslint package.json tsconfig.json \"src/**/*.ts\" \"test/**/*.ts\" --fix --fix-type [problem,suggestion]",
-    "lint": "eslint package.json tsconfig.json src --ext .ts -f html -o template-lintReport.html || exit 0",
+    "lint:fix": "eslint package.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
```

Comment:
what does eslint even check inside api-extractor?

Also I thought @willmtemple was going to deprecate our usage of eslint for json testing

Reply:
I'm leaving this up to @willmtemple

Reply:
That is pending merge of dev-tool. We are going to remove json linting and add just a script that checks our JSON files.